ibverbs-sys/src/lib.rs

```diff
@@ -48,7 +50,6 @@ pub struct ibv_wc {
     pub dlid_path_bits: u8,
 }
 
-#[cfg_attr(feature = "cargo-clippy", allow(len_without_is_empty))]
```

Comment:
Does clippy not warn on that any more? I think this should probably be
```rust
#[allow(clippy::len_without_is_empty)]
```
these days.

Reply:
Oh interesting... This might be part of a bad rebase